subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv_aux.c

```diff
@@ -433,6 +490,15 @@ static inline int isr_rx_pdu(struct lll_adv_aux *lll_aux,
 	addr = &pdu_aux->adv_ext_ind.ext_hdr_adi_adv_data[1];
 	tx_addr = pdu_aux->tx_addr;
 
+	hdr = (void *)&pdu_aux->adv_ext_ind.ext_hdr_adi_adv_data[0];
+	if (hdr->tgt_addr) {
+		/* TargetA is placed at 8th byte of ext hdr data */
+		tgt_addr = &pdu_aux->adv_ext_ind.ext_hdr_adi_adv_data[7];
```

Comment:
Consider adding `#define`s for these indexes. 

Also consider putting 
```
hdr = (void *)&pdu_aux->adv_ext_ind.ext_hdr_adi_adv_data[0];
``` 
before 
```
addr = &pdu_aux->adv_ext_ind.ext_hdr_adi_adv_data[1];
```
for semantic reasons :)

Reply:
this will be addressed once https://github.com/zephyrproject-rtos/zephyr/pull/29895 is merged